scripts/build_ngx_pagespeed.sh

```diff
@@ -420,11 +420,20 @@ function build_ngx_pagespeed() {
 
   if "$DYNAMIC_MODULE"; then
     # Check that ngx_pagespeed and nginx are recent enough to support dynamic
-    # modules.
-    if version_older_than "$NPS_VERSION" "1.10.33.5"; then
-      fail "
+    # modules.  Unfortunately NPS_VERSION might be a tag, in which case we don't
+    # know.  If it's not a numeric version number, then assume it's recent
+    # enough and if it's not they'll get an ugly compilation error later.
+    # Luckily 1.10.33.5 was a while ago now.
+    #
+    # I'd like to use bash regexps here, but they changed syntax between v3 and
+    # v4 (quotes moved from mandatory to optional to prohibited) so I can't use
```

Comment:
What do you mean by "bash rexexps"? Double-box brackets? You should be able to replace the grep with:

"${NPS_VERSION#*[^0-9.]}" != "$NPS_VERSION" 

You *could* check ${BASH_VERSINFO[0]}.